DataConnectors/Netskope/Netskope_API_FunctionApp.json

```diff
@@ -0,0 +1,128 @@
+{
+    "id": "Netskope",
+    "title": "Netskope",
+    "publisher": "Netskope",
+    "descriptionMarkdown": "The [Netskope Cloud Security Platform](https://www.netskope.com/platform) connector provides the capability to ingest Netskope logs and events into Azure Sentinel. The connector provides visibility into Netskope Platform Events and Alerts in Azure Sentinel to improve monitoring and investigation capabilities.",
+    "additionalRequirementBanner":"These queries are based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **Netskope**. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinelgithubparsersnetskope)",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Netskope",
+            "baseQuery": "Netskope_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Users",
+            "query": "Netskope\n | summarize count() by SrcUserName \n | top 10 by count_"
+        },
+        {
+            "description" : "Top 10 Alerts",
+            "query": "Netskope\n | where isnotempty(AlertName) \n | summarize count() by AlertName \n| top 10 by count_"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "Netskope_CL",
+            "lastDataReceivedQuery": "Netskope_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "Netskope_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1,
+        "isPreview": true
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "read": true,
+                    "delete": true
+                }
+            }
```

Comment:
Missing permissions - "Microsoft.OperationalInsights/workspaces/sharedKeys", Please add. Refer https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_REST_API_AzureFunctionApp_template/DataConnector_API_AzureFunctionApp_template.json

Reply:
updated and submitted.